src/plugins/xerces/README.md

```diff
@@ -2,10 +2,11 @@
 - infos/author = e1528532 <e1528532@libelektra.org>
 - infos/licence = BSD
 - infos/provides = storage/xml
+- infos/needs =
 - infos/placements = getstorage setstorage
 - infos/status = recommended experimental unittest
+- infos/metadata = elektraXercesOriginalRootName
```

Comment:
ah i see. But the issue i was thinking about is that given the following xml file:
<root rootname="attribute"></root> would cause it to be serialized as an attribute then, since rootname would be both the name of the xml attribute and the metadata for storing the original name. With it being xerces specific this would be no issue. I thought that rootname isn't such an uncommon name that this would never be the case.